docs/entities/clusterenvironment.md

```diff
@@ -78,13 +86,17 @@ status:
 
 ### Creation
 
-When a Cluster Environment is created, Primaza must verify the connection and update the field `status.state` accordingly.
+When a Cluster Environment is created, Primaza verifies the connection to the cluster and, if the connection test passes, its permissions in application and service namespaces.
```

Comment:
This sentence is not clear did you mean something like this:
When a Cluster Environment is created, Primaza verifies the connection to the cluster, whether the connection test passes,  and its permissions to application and service namespaces.

Reply:
looks good to me